tasks/Debian.yml

```diff
@@ -10,4 +10,9 @@
   apt:
     name: puppet-module-openio-openiosds
     update_cache: yes
+    
+- name: "Install the package "netcat"
```

Comment:
you should use single quotes`'` to surround the whole sentence:
`name: 'Install the package "netcat"'`